lib/helpers/deprecationWarning.js

```diff
@@ -1,4 +1,5 @@
 export default function deprecate (commandName) {
+    if (!global.browser.options.deprecationWarnings) return
```

Comment:
This wouldn't work in standalone mode. Let's call all `deprecate` functions with `this` context and access that option in that way.